programs/hash/hello.c

```diff
@@ -48,25 +48,40 @@ int main( void )
 #else
 int main( void )
 {
-    int i, ret;
+    int i, ret = MBEDTLS_EXIT_SUCCESS;
```

Comment:
Please add a variable for `exit_code` and distinguish between the return code from the functions (named `ret`) and the error code returned to the user( named `exit_code`)